chat/src/test/java/SpaceManageMessageTestIT.java

```diff
@@ -0,0 +1,172 @@
+import static org.exoplatform.platform.qa.ui.selenium.logger.Logger.info;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+
+import com.codeborne.selenide.Condition;
+
+import org.exoplatform.platform.qa.ui.commons.Base;
+import org.exoplatform.platform.qa.ui.gatein.pageobject.UserAddManagement;
+import org.exoplatform.platform.qa.ui.gatein.pageobject.UserAndGroupManagement;
+import org.exoplatform.platform.qa.ui.selenium.platform.HomePagePlatform;
+import org.exoplatform.platform.qa.ui.selenium.platform.ManageLogInOut;
+import org.exoplatform.platform.qa.ui.selenium.platform.NavigationToolbar;
+import org.exoplatform.platform.qa.ui.selenium.platform.social.SpaceHomePage;
+import org.exoplatform.platform.qa.ui.selenium.platform.social.SpaceManagement;
+import org.exoplatform.platform.qa.ui.selenium.platform.social.SpaceSettingManagement;
+
+@Tag("smoke")
+@Tag("chat")
+public class SpaceManageMessageTestIT extends Base {
+  HomePagePlatform       homePagePlatform;
+
+  NavigationToolbar      navigationToolbar;
+
+  UserAddManagement      userAddManagement;
+
+  ManageLogInOut         manageLogInOut;
+
+  UserAndGroupManagement userandgroupmanagement;
+
+  ChatManagement         chatManagement;
+
+  SpaceManagement        spaceManagement;
+
+  SpaceHomePage          spaceHomePage;
+
+  SpaceSettingManagement spaceSettingManagement;
+
+  @BeforeEach
+  public void setupBeforeMethod() {
+    info("Start setUpBeforeMethod");
+    homePagePlatform = new HomePagePlatform(this);
+    navigationToolbar = new NavigationToolbar(this);
+    userAddManagement = new UserAddManagement(this);
+    manageLogInOut = new ManageLogInOut(this);
+    userandgroupmanagement = new UserAndGroupManagement(this);
+    chatManagement = new ChatManagement(this);
+    spaceManagement = new SpaceManagement(this);
+    spaceHomePage = new SpaceHomePage(this);
+    spaceSettingManagement = new SpaceSettingManagement(this);
+  }
+
+  @Test
+  public void test01_SendMessageOnSpaceChat() {
+    String space = "space" + getRandomNumber();
+    String usernamea = "usernamea" + getRandomString();
+    String password = "123456";
+    String emaila = usernamea + "@test.com";
+    String message = "message" + getRandomNumber();
+    info("Create new user");
+    navigationToolbar.goToAddUser();
+    userAddManagement.addUser(usernamea, password, emaila, usernamea, usernamea);
+    info("Create a space");
+    homePagePlatform.goToHomePage();
+    homePagePlatform.goToMySpaces();
+    spaceManagement.addNewSpaceSimple(space, space);
+    info("connect with user ");
+    manageLogInOut.signIn(usernamea, password);
+    homePagePlatform.goToAllSpace();
+    spaceManagement.goToAllSpacesTab();
+    info("send request");
+    spaceManagement.sendARequestToASpace(space);
+    homePagePlatform.goToHomePage();
+    manageLogInOut.signIn("root", "gtn");

```

Comment:
don't use strings to sign in, if the default root user is updated, this test will fail.